ui/src/App.tsx

```diff
@@ -30,6 +31,7 @@ export const App = (): React.ReactElement => {
   const [notification, setNotification] = useRecoilState<string | undefined>(
     currentNotification,
   );
+  const [searchParams] = useSearchParams();
```

Comment:
Unless you see this query param being passed to other page components as props, I don't think this should be in the top-level App Component. I would suggest moving it to the CreateApp Component...or even maybe the AppForm component itself if possible.